lib/remote/apilistener.cpp

```diff
@@ -368,7 +368,41 @@ void ApiListener::Stop(bool runtimeDeleted)
 	m_Timer->Stop(true);
 	m_RenewOwnCertTimer->Stop(true);
 
+	m_ListenerWaitGroup->Cancel();
+	m_ListenerWaitGroup->Join();
+
+	DisconnectJsonRpcConnections();
+
+	{
+		std::unique_lock lock(m_HttpClientsLock);
+		Log(LogInformation, "ApiListener")
+			<< m_HttpClients.size() << " HTTP connections remaining.";
+	}
+
+	m_Timer->OnTimerExpired.disconnect_all_slots();
+	m_Timer->OnTimerExpired.connect([this](const Timer *const&) {
+		m_WaitGroup->Cancel();
+		DisconnectHttpClients();
```

Comment:
Also, why to disconnect them at all? IMAO they may GET stuff as long as the process is still alive.